File: ChequeApplication/Models/Cheque.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ChequeApplication.Models
{
    public class Cheque
    {
        public DateTime ChequeDate { get; set; }

        public string Payee { get; set; }

        public Double Amount { get; set; }

        public string AmountText { get; set; }
    }
}

File: Angular/src/app/models/cheque.ts
export class cheque {
  chequeDate: Date;
  payee: string;
  amount: number;
  amountWord: string;
  constructor(
    fields?: {
      chequeDate?: Date;
      payee?: string;
      amount?: number;
      amountWord?: string;
    }) {
    if (fields) Object.assign(this, fields);
  }
}
